### PR TITLE
Add Insert dataAvailable notifications for recipient

### PR DESCRIPTION
**Pull request by mbusk88**

### CODE REVIEW

File: source/Energinet.DataHub.PostOffice.Infrastructure/Documents/CosmosSequenceNumber.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+namespace Energinet.DataHub.PostOffice.Infrastructure.Documents
+{
+    // JsonProperty is required for all properties, as the object is
+    // used in CosmosDbFixture where naming rules are not yet configured,
+    public sealed record CosmosSequenceNumber
+    {
+        public CosmosSequenceNumber(long sequenceNumber)
+        {
+            Id = "1";
```
Comment (xnetsrak) [inline source/Energinet.DataHub.PostOffice.Infrastructure/Documents/CosmosSequenceNumber.cs:25]: 1  as Id ?

**Review by xnetsrak — APPROVED** [review]

File: source/Energinet.DataHub.PostOffice.Infrastructure/Documents/CosmosSequenceNumber.cs (hunk 1 above)
Comment (defectiveAi) [reply source/Energinet.DataHub.PostOffice.Infrastructure/Documents/CosmosSequenceNumber.cs:25]: We need a fixed value, as there is only a single document. Id could also be 'SequenceNumberFixedId' or a fixed GUID. Suggestions are welcome.

Comment (mbusk88) [reply source/Energinet.DataHub.PostOffice.Infrastructure/Documents/CosmosSequenceNumber.cs:25]: Yes, as there will only be one object in db where sequenceNumber will be overwritten

File: source/Energinet.DataHub.PostOffice.Application/Handlers/UpdateMaximumSequenceNumberCommandHandler.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        private readonly ISequenceNumberRepository _sequenceNumberRepository;
+
+        public UpdateMaximumSequenceNumberCommandHandler(ISequenceNumberRepository dataAvailableNotificationRepository)
+        {
+            _sequenceNumberRepository = dataAvailableNotificationRepository;
+        }
+
+        protected override async Task Handle(UpdateMaximumSequenceNumberCommand request, CancellationToken cancellationToken)
+        {
+            if (request is null)
```
Comment (michaelmajgaard) [inline source/Energinet.DataHub.PostOffice.Application/Handlers/UpdateMaximumSequenceNumberCommandHandler.cs:35]: Use `Guard.ThrowIfNul(...)`

**Review by michaelmajgaard — APPROVED** [review]

**Review by xnetsrak — APPROVED** [review]

**Pull request merged**